src/include/OpenImageIO/unordered_map_concurrent.h

```diff
@@ -127,13 +127,13 @@ class unordered_map_concurrent {
 
         // Dereferencing returns a reference to the hash table entry the
         // iterator refers to.
-        typename BinMap_t::value_type & operator* () {
+        const typename BinMap_t::value_type & operator* () const {
```

Comment:
Are you sure about this? Versus adding a second, `const` dereference? Might not some use of unordered_map_concurrent, current or future, desire a dereference that returns a reference to a mutable object?

Reply:
The API for the non-const flavor is a bit different than the STL with this particular map, so I figured I would punt on this until we needed it.

Reply:
ok, fine then